docker/docker-compose.yaml

```diff
@@ -31,11 +31,13 @@ services:
       - textile-fc-lotus:/root/lotus
   
   lotus:
-    image: textile/lotus:af1d545
+    image: textile/lotus:75041b9
     volumes:
       - textile-fc-lotus:/data
+    environment:
+      - LOTUS_API_LISTENADDRESS=/ip4/0.0.0.0/tcp/1234/http
```

Comment:
Now `textile/lotus` isn't running a reverse proxy at 0.0.0.0. By default Lotus will do on 127, but now accepts envs so we can override easily and avoid the reverse proxy.